src/pages/library.tsx

```diff
@@ -94,6 +94,7 @@ const LibraryPage = () => {
 
         playlists.forEach(playlist => {
             if (playlist.totalLength === 0) return;
+            if (!playlist.ownedBySelf && indexOfFirstNotOwned !== -1) indexOfFirstNotOwned = playlistUris.length;
```

Comment:
is this supposed to be `indexOfFirstNotOwned === -1` ?

Reply:
Yes